content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md

```diff
@@ -375,6 +371,8 @@ sensu-backend start \
 
 ## Troubleshoot clusters
 
+If you want to restart a cluster, read [Remove and redeploy a cluster][23].
```

Comment:
I want to provide an example or two of a reason to restart a cluster. Are there reasons to do this aside from more than one cluster member failing and taking the whole cluster down?

Reply:
Answered in slack: https://sensu.slack.com/archives/C60EEQFH8/p1620224379273200